### PR TITLE
Fixes #3719 - rewrite Get-Process and update examples

### LINKED ISSUE #3719: A header too many
In this example there is a &lt;TableColumnHeader/&gt; too much

&lt;View&gt;
  &lt;Name&gt;System.Globalization.CultureInfo&lt;/Name&gt;
  &lt;ViewSelectedBy&gt;
    &lt;TypeName&gt;Deserialized.System.Globalization.CultureInfo&lt;/TypeName&gt;
    &lt;TypeName&gt;System.Globalization.CultureInfo&lt;/TypeName&gt;
  &lt;/ViewSelectedBy&gt;
  &lt;TableControl&gt;
    &lt;TableHeaders&gt;
      &lt;TableColumnHeader&gt;
        &lt;Width&gt;16&lt;/Width&gt;
      &lt;/TableColumnHeader&gt;
      &lt;TableColumnHeader&gt;
        &lt;Width&gt;16&lt;/Width&gt;
      &lt;/TableColumnHeader&gt;
      &lt;TableColumnHeader/&gt;
    &lt;/TableHeaders&gt;
    &lt;TableRowEntries&gt;
      &lt;TableRowEntry&gt;
        &lt;TableColumnItems&gt;
          &lt;TableColumnItem&gt;
            &lt;PropertyName&gt;LCID&lt;/PropertyName&gt;
          &lt;/TableColumnItem&gt;
          &lt;TableColumnItem&gt;
            &lt;PropertyName&gt;Name&lt;/PropertyName&gt;
          &lt;/TableColumnItem&gt;
          &lt;TableColumnItem&gt;
            &lt;PropertyName&gt;DisplayName&lt;/PropertyName&gt;
          &lt;/TableColumnItem&gt;
        &lt;/TableColumnItems&gt;
      &lt;/TableRowEntry&gt;
    &lt;/TableRowEntries&gt;
  &lt;/TableControl&gt;
&lt;/View&gt;

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8e5d7800-2767-085a-e901-c9c85bd60111
* Version Independent ID: e363027f-b76b-edbb-22dc-02d5924cfcb9
* Content: [about_Format.ps1xml](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_format.ps1xml?view=powershell-6)
* Content Source: [reference/6/Microsoft.PowerShell.Core/About/about_Format.ps1xml.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Format.ps1xml.md)
* Product: **powershell**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [X] Impacts 6 document
- [X] Impacts 5.1 document
- [X] Impacts 5.0 document
- [X] Impacts 4.0 document
- [X] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[2d541db](https://github.com/bobbytreed/PowerShell-Docs/pull/3733/commits/2d541db673ef42cef84496d48b3f2baf309e8fdd)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[2d541db](https://github.com/bobbytreed/PowerShell-Docs/pull/3733/commits/2d541db673ef42cef84496d48b3f2baf309e8fdd)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 46 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[2d541db](https://github.com/bobbytreed/PowerShell-Docs/pull/3733/commits/2d541db673ef42cef84496d48b3f2baf309e8fdd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/3.0/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-3.0&branch=pr-en-us-3733) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/4.0/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-4.0&branch=pr-en-us-3733) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/5.0/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-5.0&branch=pr-en-us-3733) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/5.1/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-5.1&branch=pr-en-us-3733) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/6/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-6&branch=pr-en-us-3733) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C19%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902191700231208-3733%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=wdkx%2FHR9lsGpz440Up4269aeM33Ktj2OEwwAs1d8VrA%3D&st=2019-02-19T17%3A09%3A48Z&se=2019-03-22T17%3A14%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9e00afc](https://github.com/bobbytreed/PowerShell-Docs/pull/3733/commits/9e00afc9058cf8607e955da64e0884b58cf210f0)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9e00afc](https://github.com/bobbytreed/PowerShell-Docs/pull/3733/commits/9e00afc9058cf8607e955da64e0884b58cf210f0)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 47 sec(s)_

File: reference/3.0/Microsoft.PowerShell.Management/Get-Process.md [hunk 1]
```diff
@@ -7,126 +7,175 @@ online version:  http://go.microsoft.com/fwlink/?LinkID=113324
...
-	@{Label = "WS(K)"; Expression = {[int]($_.WS / 1024)}},
-	@{Label = "VM(M)"; Expression = {[int]($_.VM / 1MB)}},
-	@{Label = "CPU(s)"; Expression = {if ($_.CPU) {$_.CPU.ToString("N")}}},
-	Id, MachineName, ProcessName -Auto
+Get-Process powershell -ComputerName S1, localhost | Format-Table `
+                               @{Label = "NPM(K)"; Expression = {[int]($_.NPM / 1024)}},
+                               @{Label = "PM(K)"; Expression = {[int]($_.PM / 1024)}},
+                               @{Label = "WS(K)"; Expression = {[int]($_.WS / 1024)}},
+                               @{Label = "VM(M)"; Expression = {[int]($_.VM / 1MB)}},
+                               @{Label = "CPU(s)"; Expression = {if ($_.CPU) {$_.CPU.ToString("N")}}},
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Management/Get-Process.md]: Why so many spaces? And the last line does not line up. Change it to: 

```
Get-Process powershell -ComputerName S1, localhost | Format-Table `
    @{Label = "NPM(K)"; Expression = {[int]($_.NPM / 1024)}},
    @{Label = "PM(K)"; Expression = {[int]($_.PM / 1024)}},
    @{Label = "WS(K)"; Expression = {[int]($_.WS / 1024)}},
    @{Label = "VM(M)"; Expression = {[int]($_.VM / 1MB)}},
    @{Label = "CPU(s)"; Expression = {if ($_.CPU) {$_.CPU.ToString("N")}}},
    Id, MachineName, ProcessName -AutoSize
```

Make this change in all versions.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9e00afc](https://github.com/bobbytreed/PowerShell-Docs/pull/3733/commits/9e00afc9058cf8607e955da64e0884b58cf210f0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/3.0/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-3.0&branch=pr-en-us-3733) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/4.0/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-4.0&branch=pr-en-us-3733) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/5.0/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-5.0&branch=pr-en-us-3733) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/5.1/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-5.1&branch=pr-en-us-3733) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/6/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-6&branch=pr-en-us-3733) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C19%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902191750294360-3733%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FeUpVQYvSkE993Q2vpFNZz6M4hXAJAhncojs85zo6c8%3D&st=2019-02-19T17%3A59%3A11Z&se=2019-03-22T18%3A04%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[45f9379](https://github.com/bobbytreed/PowerShell-Docs/pull/3733/commits/45f9379b0ddfe3f96abe9c24ed469a38c44677c7)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

File: reference/3.0/Microsoft.PowerShell.Management/Get-Process.md (hunk 1 above)
Comment (bobbytreed) [reply reference/3.0/Microsoft.PowerShell.Management/Get-Process.md]: Done

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[45f9379](https://github.com/bobbytreed/PowerShell-Docs/pull/3733/commits/45f9379b0ddfe3f96abe9c24ed469a38c44677c7)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 47 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[45f9379](https://github.com/bobbytreed/PowerShell-Docs/pull/3733/commits/45f9379b0ddfe3f96abe9c24ed469a38c44677c7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/3.0/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-3.0&branch=pr-en-us-3733) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/4.0/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-4.0&branch=pr-en-us-3733) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/5.0/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-5.0&branch=pr-en-us-3733) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/5.1/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-5.1&branch=pr-en-us-3733) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3720/reference/6/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-process?view=powershell-6&branch=pr-en-us-3733) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C19%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902191810324138-3733%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=A9kh2k2fDmG4pBtr2t8wxxewoVbp%2BxsCE89gRXB6FTo%3D&st=2019-02-19T18%3A20%3A40Z&se=2019-03-22T18%3A25%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**